src/extension.ts

```diff
@@ -70,6 +70,12 @@ export function activate(context: ExtensionContext) {
       languageClient.sendNotification("quarkus/quarkusPropertiesChanged", event);
     }));
 
+    context.subscriptions.push(commands.registerCommand('quarkus/clientSettingsUpdate', (setting: string, value: any) => {
```

Comment:
there should be a flag to indicate if the preference should be added, updated, deleted